routes/index.js

```diff
@@ -36,23 +38,26 @@ module.exports = function(app) {
         // Here, we just use the host for the application making the request,
         // but you can hard code it or use something different if need be
         var url = 'http://' + request.headers.host + '/outbound';
-        
-        // Place an outbound call to the user, using the TwiML instructions
-        // from the /outbound route
-        client.makeCall({
+
+        const options = {
             to: request.body.phoneNumber,
             from: config.twilioNumber,
-            url: url
-        }, function(err, message) {
-            console.log(err);
-            if (err) {
-                response.status(500).send(err);
-            } else {
-                response.send({
-                    message: 'Thank you! We will be calling you shortly.'
-                });
-            }
-        });
+            url: url,
+        };
+
+        // Place an outbound call to the user, using the TwiML instructions
+        // from the /outbound route
+        client.api.v2010.accounts(config.accountSid).calls.create(options)
```

Comment:
Not sure why you're using this interface? Why not using something the canonical approach https://www.twilio.com/docs/api/rest/making-calls?